near/omni-bridge/src/lib.rs

```diff
@@ -627,6 +644,12 @@ impl Contract {
             )
             .into()
         } else {
+            if let OmniAddress::Btc(_) = transfer_message.recipient {
+                let token = self.get_token_id(&transfer_message.token);
+                let btc_account_id = self.get_native_token_id(ChainKind::Btc);
+                assert_eq!(token, btc_account_id);
```

Comment:
Should this be a `require!`?

Reply:
Also, does it make sense to move to `process_fin_transfer_to_other_chain`?

Reply:
fixed: https://github.com/Near-One/omni-bridge/pull/348/commits/e777e154543d5af7c2497c2ed7a20192dab20bd9